### PR TITLE
K8SPG-520 Added Persistent storage configuration description

### PR DESCRIPTION
**Pull request by nastena1606**
This Secret can store credentials for several repositories presented as
   separate data keys.

### CODE REVIEW

File: docs/backups-storage.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Configure backup storage for your [backup repositories](backups.md#backup-reposi
```
Comment (fiowro) [inline docs/backups-storage.md:4]:
```suggestion
The required configuration options depend on which cloud storage (or just Persistent Volume) you intend to use for backups:

```

Perhaps mentioning PV one more time will help the reader realize that there is a tab with it hidden at the end.

File: docs/backup-tutorial.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # 4 Make a backup
```
Comment (fiowro) [inline docs/backup-tutorial.md:2]:
```suggestion

## Set up cloud or local storage for backups

```

Having a subsection header here should improve the local TOC in the right panel

File: docs/backups-storage.md (hunk 1 above)
Comment (fiowro) [reply docs/backups-storage.md:4]: Also, what do you think about adding a subsection in the very beginning (may be on line 3), something like this:

## Set up cloud or local storage for backups

Having a subsection header should improve the local TOC in the right panel

Comment (nastena1606) [reply docs/backups-storage.md:4]: But the whole doc is titled Backup storage configuration. It makes little sense to name the subheading the same

Comment (fiowro) [reply docs/backups-storage.md:4]: Any other name? The problem I've found is on the the screenshot 
image

**Pull request merged**